CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Change Log
+All notable changes to QingStor qcamel will be documented in this file.
+
+## [v1.1.0] - 2017-09-14
+
+### Added
+
+- Support migration from Upyun to QingStor.
+
```

Comment:
We don't need to add an extra line between list items.

Reply:
And `UPyun` should be `UPYUN Storage Service`, `QingStor` should be `QingStor Object Storage`.

Below is the same.